data/STRchive-database.json

```diff
@@ -1520,9 +1520,9 @@
         "Mechanism":"Haploinsufficiency",
         "Mechanism_detail":"Reduced transcript 2",
         "Mechanism_source":"PMCID: PMC10042577",
-        "source":"https:\/\/www.omim.org\/entry\/620174, https:\/\/www.cell.com\/ajhg\/fulltext\/S0002-9297(22)00506-7, PMCID: PMC10042577, PMID: 37399286",
-        "notes":"250-300 pathogenic with incomplete penetrance, >300 complete penetrance",
-        "details":"Incomplete penetrance.",
+        "source":"https:\/\/www.omim.org\/entry\/620174, https:\/\/www.cell.com\/ajhg\/fulltext\/S0002-9297(22)00506-7, PMCID: PMC10042577, PMID: 37399286, PMID: 39227614",
+        "notes":"Flanking regions appear to correlate with repeat size (PMID: 39227614, PMID: 38937606).",
+        "details":"The 250-300 repeats range is linked to incomplete penetrance and >300 repeats with complete penetrance in some studies and resources. However, our thresholds are taken from suggestions made by Mohren et al with aggregate data (PMID: 39227614). Additionally, this study suggests that benign motifs may disrupt the formation of secondary structures in DNA/RNA, leading to reduced pathogenicity.",
```

Comment:
Could you please add the citation for the previous ranges in that sentence? And summarize what is meant by "our thresholds are taken from suggestions made by Mohren et al with aggregate data". Why did the propose the change?